src/dashboard/overview/OverviewItem.tsx

```diff
@@ -91,82 +94,67 @@ const OverviewItem = (props: OverviewItemProps): ReactElement => {
     );
   };
 
-  const isDownloadLogsEnabled = (status: Status): boolean => {
```

Comment:
By moving the _Download Logs_ button inside the expanded view, there is a problem. As the expanded view is only visible when opendexd is up and running (since the information displayed is not available otherwise), the logs should still be available, no matter what the opendexd status is. Also, the logs should be available for any service that has a container, even if there's no additional information. The easiest solution would be to leave the Download Logs button in the main view. Otherwise in some cases the view should be expanded just to see the _Download Logs_ button which is bizarre.

Reply:
I moved it back to the main view.